libsolidity/interface/CompilerStack.cpp

```diff
@@ -1244,15 +1245,16 @@ void CompilerStack::assemble(
 	if (
 		m_evmVersion >= langutil::EVMVersion::spuriousDragon() &&
 		compiledContract.runtimeObject.bytecode.size() > 0x6000
-	)
+	   )	
```

Comment:
This should be at the same level as the `if`, not `(`.
```suggestion
	)
```